hw_5/script.js

```diff
@@ -0,0 +1,90 @@
+class Entity {
+  constructor(name) {
+    this.name = name;
+  }
+
+  toString() {
+    return `Entity name: ${this.name}`;
+  }
+}
+
+class Stuff extends Entity {
+  constructor(name) {
+    super(name);
+  }
+
+  toString() {
+    return `The Stuff with name: ${this.name}`;
+  }
+}
+
+class Box extends Entity {
+  constructor(name, owner, stuff) {
+    super(name);
+    this.stuff = stuff ? stuff : [];
+    this._owner = owner;
+  }
+
+  get owner() {
+    return this._owner;
+  }
+
+  set owner(user) {
+    if (!(user instanceof User)) {
+      throw new Error(`${user} is not a User`);
+    }
+    this._owner = user;
+  }
+
+  toString() {
+    return `The Box with name: ${this.name}, owner: ${this.owner}, stuff: [${this.stuff}]`;
+  }
+
+  addStuff(item) {
+    if (item instanceof Stuff) {
+      this.stuff.push(item);
+    } else {
+      throw new Error(`The ${item} is not a Stuff`);
+    }
+  }
+}
+
+class User extends Entity {
+  constructor(name) {
+    super(name);
+  }
+
+  toString() {
+    return `{ User name: ${this.name} }`;
+  }
+}
+
+// let kanstantsin = new User("Kanstantsin");
+//
+// let kanstantinsFirstBox = new Box("box1", kanstantsin);
```

Comment:
can we use here `const `insted `let`?

Reply:
https://github.com/BlackTruth/javascript/blob/3e02ee78527ee2ff262c41e206de768fde8c4212/hw_5/script.js#L63-L67
I don't pay attention to it as always) Yes, we can, I fixed it.